pkg/event/target/nsq.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * (C) 2018 Matthias Schneider
```

Comment:
The copyright should be `Minio Cloud Storage 2018` we do not accept personal copyright headers